### ISSUE TITLE
Project Doesn't run, missing framework file, I think

### ISSUE THREAD

**Issue by msdickinson** [asu-ke-web-services/dont-get-fired-game#121]
Either there is a file missing, like the error msg is suggesting.
I have attempted to find this missing file, and have had no luck.
I have also looked up the error msg as well, with no luck

![error missing file](https://cloud.githubusercontent.com/assets/2480828/13836247/8977b7f8-ebbc-11e5-9a07-beefb9d45504.PNG)

![more](https://cloud.githubusercontent.com/assets/2480828/13836367/c495bdd4-ebbd-11e5-8dd1-6cdd84e5be8b.PNG)

It must have happened when i pulled merged with the latest. I know mine ran before that.

This is the the current verison. Tuyen said hes is showing same issue.

**Comment by msdickinson** [comment]
this seems related... not sure

https://github.com/airbnb/enzyme/issues/186

**Comment by rraub** [comment]
I am not having any problems running the tests or the game on the current dev branch. Did you try `npm install` and then restart meteor? If that doesn't fix it, then try and re-clone this repository and `npm install` everything again. Are both you and Tuyen on Windows? 

Its most likely that you got in some weird state somehow or given that we are running on beta versions of some packages we may need to update them in the future.

**Comment by msdickinson** [comment]
well, I have found on my home machine. that if i delete everything and re pull it down it fixs it some times. I can try that on my home machine. I know tyuen has hes laptop and hes doesnt run. so maybe he can try that before the meeting. so we will know if it works when we get there. 

I did try npm install 
and the tests all pass. we are both on windows

Maybe David can pull the latest and see if it works on hes.
